doc/run_experiment.rst

```diff
@@ -595,11 +595,15 @@ fixed_parameters *(Optional)*
 """""""""""""""""""""""""""""
 
 List of dictionaries containing parameters you want to have fixed for each
-learner in :ref:`learners` list. Any empty ones will be ignored
-(and the defaults will be used). If :ref:`grid_search` is ``True``,
+learner in the :ref:`learners` list. Empty dictionaries will be ignored
+and the defaults will be used for these learners. If :ref:`grid_search<grid_search>` is ``True``,
 there is a potential for conflict with specified/default parameter grids
 and fixed parameters.
 
+.. note::
+    Tuples are not supported in the config file, and will lead to parsing errors.
+    Make sure to replace tuples with lists when specifying fixed parameters.
+
```

Comment:
Can we add an example here? I think I had one in the issue?

Reply:
I added it in the beginning and then thought it was too lengthy and that whoever made it this far will know what a tuple is :)
I don't mind putting it back.

Reply:
I think it's more to show folks what kinds of parameters are usually tuples. I think it's worth putting it back in.